Add console dot info, warn and error documentation (#107174)
Azure/azure-content: articles/stream-analytics/stream-analytics-javascript-user-defined-functions.md

```

## User Logging

> The logging mechanism allow users to capture custom information while a job is running. Log data can be used to debug or assess the correctness of the custom code in real time. This mechanism is available through 3 different methods.

Reviewer: ```suggestion
The logging mechanism allows users to capture custom information while a job is running. Log data can be used to debug or assess the correctness of the custom code in real time. This mechanism is available through three different methods.
```
Minor edits for grammar and Microsoft style

### Console.Info()
Console.Info method is used to log general information during code execution. This method will log data without interrupting computation. The message logged will be marked as Event Level Informational.

```javascript
console.info('my info message');
```

### Console.Warn()

> Console.Warn method is used to log data that may not be correct or expected but are still accepted for computation. This method will not interrupt computation and will resume running after method is return. The message logged will be marked as Event Level Warning.

Reviewer: ```suggestion
Console.Warn method is used to log data that might not be correct or expected but is still accepted for computation. This method will not interrupt computation and will resume running after the method is returned. The message logged will be marked as Event Level Warning.
```
Minor grammar edits

```javascript
console.warn('my warning message');
```

### Console.Error() and Console.Log()

> Console.Error method is only used to log error case where code cannot continue to run. This method will throw exception with the error information provided as input parameter and job will stop running. The error message logged will be marked as Event Level Error.

Reviewer: ```suggestion
Console.Error method is only used to log error cases where code cannot continue to run. This method will throw an exception with the error information provided as the input parameter and job will stop running. The error message logged will be marked as Event Level Error.
```
Minor edits for grammar
